### PR TITLE
Feature - add Azure Blob Container dependency tracking

### PR DESCRIPTION
**Pull request by stijnmoreels**
Relates to #56

### CODE REVIEW

File: src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs [hunk 1]
```diff
@@ -314,6 +314,46 @@ public static void LogServiceBusDependency(this ILogger logger, string entityNam
...
+        ///     Logs an Azure Blob Storage Dependency.
+        /// </summary>
+        /// <param name="logger">Logger to use</param>
+        /// <param name="accountName">Account of the storage resource</param>
+        /// <param name="blobName">Name of the Blob Container resource</param>
+        /// <param name="isSuccessful">Indication whether or not the operation was successful</param>
+        /// <param name="startTime">Point in time when the interaction with the dependency was started</param>
+        /// <param name="duration">Duration of the operation</param>
+        /// <param name="context">Context that provides more insights on the dependency that was measured</param>
+        public static void LogBlobStorageDependency(this ILogger logger, string accountName, string blobName, bool isSuccessful, DateTimeOffset startTime, TimeSpan duration, Dictionary<string, object> context = null)
```
Comment (tomkerkhove) [inline src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs:346]: Not sure what they expect but I would change this to the container name instead

**Comment by tomkerkhove** [comment]
I don't see it pop up on the App Map or is it just me? 🤔

**Comment by stijnmoreels** [comment]
> I don't see it pop up on the App Map or is it just me? 🤔

image

The start time is randomized till 24h, so you got to change the timespan in which the app map is shown.

**Comment by tomkerkhove** [comment]
Ok thanks!

**Pull request merged**